src/main/scala/edu/cmu/cs/obsidian/Main.scala

```
@@ -287,27 +287,28 @@ object Main {
             for (p <- protobufs) {
                 val protobuf = p._1
                 val filename = p._2
+                if(filename.length() > 0) {
```

Shouldn't we report an error if this is not the case?

---

Removed code generation for utility classes